test/unit/rules/lint-require-valid-alt-text-test.js

```diff
@@ -0,0 +1,128 @@
+'use strict';
+
+const generateRuleTests = require('../../helpers/rule-test-harness');
+
+generateRuleTests({
+  name: 'require-valid-alt-text',
+
+  config: true,
+
+  good: [
+    '<img alt="hullo">',
```

Comment:
For this to be a useful example, it should mimic meaningful text that would also be useful- like a description of something that could be purchased online, perhaps?

`<img alt="a shirt with the ember logo across the front">`